climbing_gym/demo/sale_order_line.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="sale_order_line_8" model="sale.order.line">
+        <field name="order_id" ref="sale_order_7"/>
+        <field name="product_id" ref="product_product_16"/>
+        <field name="name"><![CDATA[1 Entrance Happy Hour]]></field>
+        <field name="product_uom_id" ref="uom.product_uom_unit"/>
+        <field name="price_unit">12.0</field>
+        <field name="technical_price_unit">12.0</field>
+        <field name="discount">100.0</field>
+    </record>
+    <record id="sale_order_line_6" model="sale.order.line">
+        <field name="order_id" ref="sale_order_6"/>
+        <field name="product_id" ref="product_product_17"/>
+        <field name="name"><![CDATA[1 Entrance Unlimited]]></field>
+        <field name="product_uom_id" ref="uom.product_uom_unit"/>
+        <field name="price_unit">15.0</field>
+        <field name="technical_price_unit">15.0</field>
+    </record>
+    <record id="sale_order_line_7" model="sale.order.line">
+        <field name="order_id" ref="sale_order_6"/>
+        <field name="product_id" ref="product_product_17"/>
+        <field name="sequence">11</field>
+        <field name="name"><![CDATA[Discount 100% on 1 Entrance Unlimited]]></field>
+        <field name="product_uom_id" ref="uom.product_uom_unit"/>
+        <field name="price_unit">-15.0</field>
+        <field name="technical_price_unit">-15.0</field>
+        <field name="reward_id" ref="loyalty_reward_2"/>
+        <field name="reward_identifier_code">2058137271</field>
```

Comment:
Does this mean anything in a user db? shouldn't we apply the loyalty reward with functions?